src/patterns/complete-multiple-tasks/index.md

```diff
@@ -135,11 +135,11 @@ Sometimes, it’s better to let the user decide when a task is completed.
 This can be helpful when a task involves:
 
 - some questions that are optional
-- writing a long answer (such as in a [textarea](/components/textarea/))
+- writing a long answer (such as in a [Textarea component](/components/textarea/))
 - looking up information, such as details about previous jobs
 - answers that need to be checked carefully with someone else
 
-Do this by asking a radio question at the end of the task – either as the last question (if the task is a single page) or on the [‘Check answers’ page](/patterns/check-answers/) (if the task uses multiple [question pages](/patterns/question-pages/)).
+Do this by asking a radio question at the end of the task – either as the last question (if the task is a single page) or on the [Check answers page in your service](/patterns/check-answers/) (if the task uses multiple [Question pages in your service](/patterns/question-pages/)).
```

Comment:
I think these are less likely to be confusing, so simplified the link text and removed the link to Question pages, which seems slightly superfluous.

```suggestion
Do this by asking a radio question at the end of the task – either as the last question (if the task is a single page) or on the [Check answers page](/patterns/check-answers/) (if the task involves more than one pages).
```